File: Exercicio2.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"");
    int raio, area;

    printf("Insira o valor do raio: ");
    scanf("%d",&raio);

    area = 3.14159 * (raio*raio);

    printf("\nO valor da área do círculo é %4.d\n\n",area);
    return 0;
}

File: Exercicio2.cs
using System;

class Program {
  public static void Main (string[] args)
        {
            double raio = 0.0, area = 0.0;

            Console.WriteLine("Insira o valor do raio: ");
            raio = Double.Parse(Console.ReadLine());

            area = 3.14159 * (raio * raio);

            Console.WriteLine("\r\nO valor da área do círculo é " + Math.Round(area, 2).ToString());
        }
}